src/pudl/metadata/dfs.py

```diff
@@ -231,3 +233,99 @@
 From FERC Form 1 page 219, Accumulated Provision for Depreciation of electric
 utility plant (Account 108).
 """
+
+POLITICAL_SUBDIVISIONS: pd.DataFrame = pd.read_csv(
+    StringIO(
+        """
+subdivision_code,subdivision_name,country_code,country_name,subdivision_type,timezone,state_id_fips,division_name_us_census,division_code_us_census,region_name_us_census,is_epacems_state
```

Comment:
Sure. I almost did this the first time around and then backed off because it seemed a little duplicative, but I agree it'd be a bad thing to get confused about. I'll switch it to `timezone_approx` and explain in the description.